src/config/dark-sites.config

```diff
@@ -524,6 +525,7 @@ niebezpiecznik.pl
 nitter.42l.fr
 nitter.dark.fail
 nitter.fdn.fr
+nitter.kavin.rocks
```

Comment:
Subdomains doesn't have to be listed.

Reply:
Why's this? I don't want all my subdomains to be listed (since not all of them are compliant, for being listed globally)
I just want the root of my domain and specific subdomains

Reply:
Because currently matching the subdomain to root domain is the behavior, see #373 #660

Reply:
I see, according to https://github.com/darkreader/darkreader/pull/830#issuecomment-439964977, I can't add a rule for this either :/

I guess I will remove the root domain for now.

Reply:
Yeah it's a problem that I still want to solve when I can find enough time and can put some better thought into it, but almost feels like solving an P = NP problem with the amount of feedback that I have gathered by now.